src/pages/iou/SplitBillDetailsPage.js

```diff
@@ -65,7 +65,7 @@ function getReportID(route) {
 
 function SplitBillDetailsPage(props) {
     const reportAction = props.reportActions[`${props.route.params.reportActionID.toString()}`];
-    const participantAccountIDs = reportAction.originalMessage.participantAccountIDs || PersonalDetailsUtils.getAccountIDsByLogins(reportAction.originalMessage.participants);
+    const participantAccountIDs = reportAction.originalMessage.participantAccountIDs;
```

Comment:
@puneetlath I thinkkkk we're only going to be able to do this once we finish the DB migration - https://github.com/Expensify/Expensify/issues/292074

Reply:
I think we should go ahead and let old IOUs break. We should clean them up, but we don't need to block this on it.

Reply:
Alright i think that's fair - then let's do this!! 👍  And the DB migration is ready to go at a moment's notice anyway so we can get this to production & finish the migration quick 👍